apps/wpcom-block-editor/src/wpcom/features/live-preview/upgrade-notice.tsx

```diff
@@ -70,14 +107,58 @@ export const LivePreviewUpgradeNotice: FC< {
 		} );
 		return () => removeNotice( UPGRADE_NOTICE_ID );
 	}, [
+		canvasMode,
 		createWarningNotice,
+		dashboardLink,
+		noticeText,
+		previewingTheme.name,
+		previewingTheme.type,
+		previewingTheme.typeDisplay,
 		removeNotice,
 		siteEditorStore,
 		upgradePlan,
-		previewingTheme.type,
-		previewingTheme.name,
-		dashboardLink,
-		previewingTheme.typeDisplay,
 	] );
+
+	/**
+	 * Show the notice when the canvas mode is 'view'.
```

Comment:
Sorry, but could you also patch this PR here as well?

- https://github.com/Automattic/wp-calypso/pull/84646

I think I somehow forgot to apply this here. Or I think there was conflict with the refactor and I just forgot to recheck 🙏

Reply:
I tried https://github.com/Automattic/wp-calypso/pull/84676/commits/80f06a987e4fd30f998fd3da9615003699edc9c5, WDYT?

Reply:
This block of code is repeated 4 times (!)

```
		if ( ! siteEditorStore ) {
			return;
		}
		if ( ! previewingTheme.name ) {
			return;
		}
```

I'm still thinking how to "refactor" it...

Reply:
(Other than that, it works 👍 )